client/src/Movies/MovieCard.js

```diff
@@ -1,7 +1,25 @@
 import React from 'react';
 
-const MovieCard = props => {
-  return;
+const MovieCard = (props) => {
+    const { title, director, metascore, stars } = props.movie;
```

Comment:
What might make even more sense here is destructuring `movie` off of `props` in the param list.
For example:
```javascript
const MovieCard = ({ movie }) => {
  const { title, director, metascore, stars } = movie;
  // ...
```